src_py/apiServer/transmitter.py

```diff
@@ -53,9 +53,9 @@ def startCasting(self, phase): # numOfBatches, is no. of batches to request from
         print('\nStart Casting Phase')
 
         if (phase=="Training"):
-            batchesPerSource = globe.expFlow["Batches per source"]["Training"]
+            batchesPerSource = globe.experiment_flow_global.expFlow["Batches per source"]["Training"]
```

Comment:
Add definition in global to "Batches per source as well" 
E.g. 
BATHCHES_PER_SOURCE_STR = "Batches per source